core/editor/config-providers/editor-v1-config-provider.php

```diff
@@ -33,10 +36,20 @@ public function get_style_configs() {
 	}
 
 	public function get_style_handles_to_enqueue() {
-		return [ 'elementor-editor' ];
+		return [
+			'elementor-editor',
+			'elementor-responsive-bar',
+		];
 	}
 
 	public function get_template_body_file_path() {
 		return __DIR__ . '/../templates/editor-body-v1.view.php';
 	}
+
+	public function get_additional_template_names() {
```

Comment:
I think it will be better to stick to one convention and maybe return paths and not names, and maybe find better name like `get_templates_paths` and maybe change the name of `get_template_body_file_path` to `get_root_template_path`

Reply:
Done